migrate-from-vitess.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Migrate Data from Vitess to TiDB
+summary: Learn about the tools to migrate data from Vitess to TiDB. 
+---
+
+# Migrate Data from Vitess to TiDB
+
+This document describes the tools that you can use to migrate data from [Vitess](https://vitess.io/) to TiDB. 
+
+As the backend of Vitess is based on MySQL, when migrating data from Vitess to TiDB, you can use the same migration tools that are applicable to MySQL, such as [Dumpling](/dumpling-overview.md), [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md), and [TiDB Data Migration (DM)](/dm/dm-overview.md). Note that these tools should be set up for each shard in Vitess for data migration.
```

Comment:
Here, it will be more appropriate to introduce to the users how to choose different solutions in different scenarios.
Since DM has already integrated Lightning physical mode since v7.6 (https://docs.pingcap.com/tidb/stable/task-configuration-file-full), we can encourage the user to use DM instead of Dumpling+Lightning+DM (for incremental data migration), it will be more easy and convenient. However, when the size of the data source is more than 10 TiB, it is recommended to use Dumpling + Lightning parallel physical mode+DM(incremental data migration) which is faster and more mature.

Reply:
Rest LGTM

Reply:
Updated in a1127d2cd1d4524ed5745be1c3d59c6a0185aa96, PTAL